### PR TITLE
Code update and unit tests

### PR DESCRIPTION
**Pull request by ydeshayes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,24 +2,16 @@
 
 Helper for running nodejs applications on Platform.sh
 
-##Purpose
+## Purpose
 
-Reads Platform.sh configuration from environment and returns a single object
+Reads [Platform.sh configuration](https://docs.platform.sh/development/variables.html#) from environment and returns a single object
```
Comment (ralt) [inline README.md]: 1. You can remove the trailing hash at the end of the URL.
2. Can you add a dot?

File: src/platformsh.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/**
+* Read number of CPUs from environment or fallback to the _private_ configuration property
+* Useful for determining the number of processes to fork.
+*/
+const num_of_cpus = () => {
```
Comment (ralt) [inline src/platformsh.js]: Errr what's the reason for using `const foo = () => {}` over `function foo () {}`?

File: src/platformsh.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  try {
+    if(process.env['OMP_NUM_THREADS']) {
+      return process.env['OMP_NUM_THREADS'];
+    }
+    const config = require('/run/config.json');
```
Comment (ralt) [inline src/platformsh.js]: Can we read this file at the top-level, to make sure it's not blocking during e.g. an HTTP request?

Comment (ydeshayes) [reply src/platformsh.js]: I thought this file could change during runtime ?

File: src/platformsh.js (hunk 2 above)
Comment (ydeshayes) [reply src/platformsh.js]: More clear to me, just ES6 style

Comment (ralt) [reply src/platformsh.js]: Let's agree to disagree :)

The `const` declaration still has the drawback of hoisting, which can lead to obscure bugs, so I still prefer `function`.

File: src/platformsh.js (hunk 3 above)
Comment (ralt) [reply src/platformsh.js]: It can't.

**Pull request merged**